label_studio/templates/base.html

```diff
@@ -101,26 +101,86 @@
 </template>
 
 <script id="app-settings" nonce="{{request.csp_nonce}}">
+
+  var __customHotkeys = {{ user.custom_hotkeys|json_dumps_ensure_ascii|safe }};
+
+  // Filter custom hotkeys for editor-specific ones
+  var editorCustomHotkeys = {};
+  var prefixRegex = /^(annotation|timeseries|audio|regions|video|image_gallery|tools):(.*)/;
+
+  for (let key in __customHotkeys) {
+      const match = key.match(prefixRegex);
+      if (match) {
+          const [, prefix, shortKey] = match;
+          
+          // Get the current value
+          const value = __customHotkeys[key];
+          
+          // Check if value has active property set to false
+          if (value && value.active === false) {
+              // Create a copy of the value with key set to null
+              const modifiedValue = {...value, key: null};
+              editorCustomHotkeys[shortKey] = modifiedValue;
+          } else {
+              // Use the original value
+              editorCustomHotkeys[shortKey] = value;
+          }
+      }
+  }
+
+  function lookupHotkey (lookup) {
+      // Check if custom hotkeys exist and the lookup is in there
+      if (window.APP_SETTINGS?.user?.customHotkeys && lookup in window.APP_SETTINGS.user.customHotkeys) {
+          const hotkeyValue = window.APP_SETTINGS.user.customHotkeys[lookup];
+          
+          // If active is explicitly false, return a copy with key set to null
+          if (hotkeyValue && hotkeyValue.active === false) {
+              return {...hotkeyValue, key: null};
```

Comment:
Would be great to have README file describing general principles of hotkeys we use. For example questions regarding this part:
- what does `active: false` mean? when do we have it?
- what does `key: null` mean? what do we do with such hotkey?